release-notes/opendistro-for-elasticsearch-release-notes-1.10.1.md

```diff
@@ -17,6 +17,12 @@ The release consists of Apache 2 licensed Elasticsearch version 7.9.1, and Kiban
 
 Open Distro for Elasticsearch 1.10.1 includes the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
 
+## BREAKING CHANGES
+
+### Release Engineering
+
+* Kibana has new cookie settings for security Kibana plugin 2.0 framework ([#397](https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/397))
+  Please follow the instructions [here](https://opendistro.github.io/for-elasticsearch-docs/docs/upgrade/1-10-1/) if you are upgrading from previous Kibana versions to 1.10.1.
```

Comment:
Replace **here** with **to upgrade the Kibana to 1.10.1**
```suggestion
  Please follow the instructions [here](https://opendistro.github.io/for-elasticsearch-docs/docs/upgrade/1-10-1/) if you are upgrading from previous Kibana versions to 1.10.1.
```

Reply:
Resolved